coords/chandra.py

```diff
@@ -18,17 +18,93 @@
 #  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #
 
-# KJG: 2016-09013 no changes needed for P3
-
-
-"""
-Routines related specifically to converting to/from Chandra coordinates
+"""Routines related specifically to converting to/from Chandra coordinates.
+
+The conversion between the various coordinate systems on Chandra
+depend on the aspect solution, since this determines where the
+spacecraft is pointing at any point during the observation. However,
```

Comment:
```suggestion
spacecraft is pointing at any given time during the observation. However,
```
"pointing at any point" sounds somewhat funny, but might be confusing to non-native speakers to have "point" once be a spatial "pointing" and once be a "point in time" within in the same sentence.